app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -65,27 +65,29 @@ keywords:
   - Settlement Scheme
 ---
 
-The information on this page is for:
+Most non-UK citizens will not be eligible for financial support from the UK government to help with teacher training.
 
-* non-UK citizens living in the UK
-* non-UK citizens living outside the UK
-* Irish citizens
+Some support may be available if you:
```

Comment:
does this need a `(one of the following)` to emphasise you don't need all of these?